server/.env.example

```diff
@@ -11,5 +11,4 @@ AUTH0_DOMAIN="auth0_domain"
 AUTH0_AUDIENCE="auth0_audience"
 AUTH0_CLIENT_ID="auth0_client_id"
 AUTH0_CALLBACK_URL=${AUTH0_AUDIENCE}/callback
-GROQ_API_KEY="your_groq_api_key"
-GROQ_TRANSCRIPTION_MODEL="whisper-large-v3"
```

Comment:
just writing a reminder so we don't forget -- need to delete this from the env variables.